package.json

```diff
@@ -87,14 +87,14 @@
     "screwdriver-build-bookend": "^2.0.1",
     "screwdriver-command-validator": "^1.0.2",
     "screwdriver-config-parser": "^3.13.0",
-    "screwdriver-data-schema": "^18.12.5",
+    "screwdriver-data-schema": "^18.13.1",
     "screwdriver-datastore-sequelize": "^4.1.1",
-    "screwdriver-executor-docker": "^2.3.1",
-    "screwdriver-executor-k8s": "^10.6.1",
+    "screwdriver-executor-docker": "^2.3.2",
+    "screwdriver-executor-k8s": "^10.7.2",
     "screwdriver-executor-k8s-vm": "^1.0.0",
     "screwdriver-executor-queue": "^1.1.0",
     "screwdriver-executor-router": "^1.0.0",
-    "screwdriver-models": "^26.8.4",
+    "screwdriver-models": "^26.8.6",
     "screwdriver-notifications-email": "^1.1.2",
     "screwdriver-notifications-slack": "^2.1.2",
     "screwdriver-scm-github": "^5.3.2",
```

Comment:
Will need to bump this to `^6.0.0`

Reply:
I want this change to be independent of the models. It can pass in the extra param without models doing anything with it. Then after confirming this doesn't break anything, we can pull in the new model

Reply:
Talking about `screwdriver-scm-github`

Reply:
Apparently `scm-github` was not bumped major. Keeping it as `5.3` now